lib/question.rb

```diff
@@ -16,6 +16,15 @@ def ask_non_whitespaces(question, field_name)
   end
 
   def ask_base(question)
-    ask("[?] #{question}")
+    ask("[?] #{question}") do |q|
+      yield(q) if block_given?
+    end
+  end
+
+  def required_with_min_length(question, field_name, min_length = 10)
```

Comment:
it is kind of defining where you wanna do the validation... I think the best way is on the Zappifest but I would not go to court with you :)